examples/all-clusters-app/esp32/main/component.mk

```diff
@@ -26,6 +26,7 @@ COMPONENT_SRCDIRS :=
   ../third_party/connectedhomeip/examples/all-clusters-app/all-clusters-common/gen  \
   ../third_party/connectedhomeip/src/app/server                                     \
   ../third_party/connectedhomeip/src/app/util                                       \
+  ../third_party/connectedhomeip/src/app/clusters/descriptor                        \
```

Comment:
Hi @vivien-apple Once you are done with all your changes, can you please rebase to latest CHIP master and add the same line in ` examples/all-clusters-app/esp32/main/CMakeLists.txt`. CMake support for ESP32 in CHIP was added today. Please see here: https://github.com/project-chip/connectedhomeip/pull/5254/files#diff-44c5e428c9e3f61eb7e4f7395385fba994e42c9e4344f84e20bbe273f3406bbbR24